airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/bump_version/pipeline.py

```diff
@@ -79,202 +60,11 @@ async def _cleanup(self) -> StepResult:
         return StepResult(step=self, status=StepStatus.SUCCESS)
 
 
-class AddChangelogEntry(Step):
```

Comment:
Moved this to `pipelines.airbyte_ci.steps.changelog` as it's a step reused between multiple commands.